### ISSUE TITLE
itty-durable require itty-router-extras butitty-router-extras advertised itself as unmaintaned since itty-router v4

### ISSUE THREAD

**Issue by wighawag** [kwhitley/itty-durable#35]
itty-durable require itty-router-extras butitty-router-extras advertised itself as unmaintaned since itty-router v4

Error:

```
Could not resolve "itty-router-extras"

    ../node_modules/.pnpm/itty-durable@1.6.0/node_modules/itty-durable/itty-durable.js:25:37:
      25 │ const itty_router_extras_1 = require("itty-router-extras");
         ╵                                      ~~~~~~~~~~~~~~~~~~~~

  You can mark the path "itty-router-extras" as external to exclude it from the bundle, which will remove this error. You can also surround this "require" call with a try/catch block to handle this failure at run-time instead of bundle-time.
```

**Comment by wighawag** [comment]
Actually, latest`itty-router-extras` do not contains any code

need to install `0.4.2` to make it work but would be great to remove the dependency

**Comment by kwhitley** [comment]
> Actually, latest`itty-router-extras` do not contains any code

Yeah, that was a major catch - thanks! 

- [x] Broken deployment fixed in `itty-router-extras @ 0.4.5`.
- [x] Published `itty-durable @ 2.0.0`, including peer dependency update to `itty-router @ ^4.0.6`
